### PR TITLE
doc/install/containers: note vX.Y.Z[-YYYYMMDD] tags

### PR DESCRIPTION
**Pull request by liewegas**

### CODE REVIEW

**Comment by liewegas** [comment]
jenkins render docs

File: doc/install/containers.rst [hunk 1]
```diff
@@ -16,13 +16,17 @@ ceph/ceph
   dependencies installed.
 - TODO: add vRELNUM.2.DOT so we have a container for every point release.
```
Comment (dsavineau) [inline doc/install/containers.rst]: Do we still need this statement since we have vRELNUM.Y.Z now ?

**Comment by liewegas** [comment]
jenkins render docs

File: doc/install/containers.rst (hunk 1 above)
Comment (liewegas) [reply doc/install/containers.rst]: removed!

**Pull request merged**